conda_build/index.py

```diff
@@ -38,28 +36,20 @@
 import conda_package_handling.api
 from conda_package_handling.api import InvalidArchiveError
 
-
-from concurrent.futures import ProcessPoolExecutor, as_completed
-from concurrent.futures import Future, Executor
-from threading import Lock
+from concurrent.futures import ProcessPoolExecutor
+from concurrent.futures import Executor
 
 #  BAD BAD BAD - conda internals
 from conda.core.subdir_data import SubdirData
 from conda.models.channel import Channel
 
-
 from . import conda_interface, utils
 from .conda_interface import MatchSpec, VersionOrder, human_bytes, context
 from .conda_interface import CondaError, CondaHTTPError, get_index, url_path
 from .conda_interface import TemporaryDirectory
 from .conda_interface import Resolve
 from .utils import glob, get_logger, FileNotFoundError, JSONDecodeError
 
-# try:
```

Comment:
Did flake8 really complain about these commented lines? It did not for me—and I would suggest we shouldn't remove it without deliberate intent. From the blame it seems @msarahan put this in here deliberately, as a reminder for future development. I'd like to recommend this be restored.

Reply:
Flake8 did not complain about it indeed. 
However, I just do not see the necessity of commenting code, since you can create an issue an relate to it and explain better why it should change and how it should be changed. From my experience (and according to the literature), those commented codes stay in the project almost forever (for example that one is 6+ month old, I have seen other older than that). I don't mind to revert it. It is just that I usually try to follow the principles of clean code. But that is fine for me, I can revert it then. :)

Reply:
Thanks. I generally agree with your position, but since neither of us wrote this code, and because the stated intent of this PR is simply to fix flake8 issues, it's better for now to avoid.

Reply:
Sure thing! No problem from my side. I already reverted it! :)

Reply:
Thanks! Will approve as soon as we're green